addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/StationHandler.java

```diff
@@ -108,6 +108,11 @@ public void dispose() {
         super.dispose();
     }
 
+    @Override
+    public void updateStatus(ThingStatus status) {
```

Comment:
As far as I understand that method can't be called from the outside ( from a thing to update the bridge [as in the unchanged code] and vice versa.